pyproject.toml

```diff
@@ -0,0 +1,85 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "sv-yosys"
+dynamic = ["version"]
+description = "Yosys wrappers for SIEVE ZK-circuit synthesis."
+readme = "README.md"
+license = { file = "LICENSE" }
+authors = [
+    { name = "Trail of Bits", email = "opensource@trailofbits.com" },
```

Comment:
Feel free to add your company address as an author as well; this is just the default.